config/users.yml

```diff
@@ -21,12 +16,7 @@
 
 - email: "mohamed.deerow@digital.cabinet-office.gov.uk"
   origin: "google"
-  deploy_envs: ['modeerow']
-  cf_admin: true
-
-- email: "rafal.proszowski@digital.cabinet-office.gov.uk"
```

Comment:
Sad times... I already miss you.

I shall presume it will not remove the account?

Reply:
It will delete the account https://github.com/alphagov/paas-cf/blob/master/scripts/sync-admin-users.rb#L38